Formula/tflint.rb

```diff
@@ -1,9 +1,9 @@
 class Tflint < Formula
   desc "Linter for Terraform files"
   homepage "https://github.com/wata727/tflint"
-  url "https://github.com/wata727/tflint/archive/v0.9.3.tar.gz"
-  sha256 "cad7d122178175eff12dad349225b4feeb02e34e0f5c021d0d73cb694b04ceb2"
-  head "https://github.com/wata727/tflint.git"
```

Comment:
Should we keep the `head`?

Reply:
I dont think so. We dont use `head` in the formula.

Reply:
If the HEAD build does not break, we probably should not remove it, even though we generally do not want to add `head` for new formulae.

Reply:
Ok. I will add it back then.

Reply:
Updated. :)

Reply:
Thanks!